test/tests/integration/measurement/timeout-result.test.ts

```diff
@@ -1,128 +0,0 @@
-import request, { type Agent } from 'supertest';
```

Comment:
Same here - why removed?

Reply:
This test was unstable since the time it was created, because of the complex awaiting logic. Now that functionality is covered with e2e test (https://github.com/jsdelivr/globalping/blob/master/test/e2e/cases/offline-probes.test.ts#L40) so that one is no longer needed.